Dockerfile

```diff
@@ -1,4 +1,15 @@
-ARG IMAGE=python:3.9-alpine
+ARG IMAGE=alpine:3.17
+FROM ${IMAGE} as builder
+
+WORKDIR /build
+
+# Install dependencies
+RUN apk add --no-cache git python3 python3-dev py3-pip py3-wheel build-base gcc libffi-dev openssl-dev musl-dev cargo
+
+# Fetch requirements
+COPY requirements.txt .
+RUN pip install -r requirements.txt
```

Comment:
Since we have moved from a `requirements.txt` to a `pyproject.toml` to describe the dependencies, these step no longer work. 
I propose #1669 as an alternative.